.github/workflows/docker.yml

```
@@ -0,0 +1,43 @@
+        uses: docker/login-action@v3.3.0
+        with:
+          registry: ${{ env.DOCKER_REGISTRY }}
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ env.DOCKER_PASSWORD }}
+
+      - name: Build and push Docker image
+        env:
+          IMG: ${{ env.DOCKER_REGISTRY }}/${{ github.repository }}:${{ github.ref_name }}
+        run: |
+          IMG_INPUT=${{ env.IMG }}
+          IMG=${IMG_INPUT,,} make docker-buildx
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)